pkg/ipamclaimswebhook/podmutator_test.go

```diff
@@ -105,11 +105,10 @@ var _ = Describe("KubeVirt IPAM launcher pod mutato machine", Serial, func() {
 
 		ipamClaimsManager := NewIPAMClaimsValet(mgr)
 
-		Expect(
-			ipamClaimsManager.Handle(context.Background(), podAdmissionRequest(config.inputPod)),
-		).To(
-			Equal(config.expectedAdmissionResponse),
-		)
+		result := ipamClaimsManager.Handle(context.Background(), podAdmissionRequest(config.inputPod))
+
+		Expect(result.Patches).To(ConsistOf(config.expectedAdmissionResponse.Patches))
+		Expect(result.AdmissionResponse).To(Equal(config.expectedAdmissionResponse.AdmissionResponse))
```

Comment:
maybe switch the order of these two ? 
@qinqon you said something about the order not mattering only for a *subset* of the tests. Could you elaborate ? 

Can this PR be merged ?

Reply:
> maybe switch the order of these two ?

the only benefit is fail fast if the AdmissionResponse is different
i think it is a nit but switched (done)

imho the PR can be merged, but maybe you wait for Quique